controller/array_action/array_mediator_svc.py

```diff
@@ -80,16 +95,8 @@ def build_kwargs_from_parameters(space_efficiency, pool_name, volume_name,
         'size': volume_size,
         'pool': pool_name
     })
-    # if space efficiency == None, create default space efficiency volume thick
-    if space_efficiency:
-        space_efficiency = space_efficiency.lower()
-        if space_efficiency == config.SPACE_EFFICIENCY_THIN:
-            cli_kwargs.update({'thin': True})
-        elif space_efficiency == config.SPACE_EFFICIENCY_COMPRESSED:
-            cli_kwargs.update({'compressed': True})
-        elif space_efficiency == config.SPACE_EFFICIENCY_DEDUPLICATED:
-            cli_kwargs.update({'compressed': True, 'deduplicated': True})
-
+    array_space_efficiency = _get_array_space_efficiency(space_efficiency)
```

Comment:
```suggestion
    space_efficiency_kwargs = _get_space_efficiency_kwargs(space_efficiency)
```

Reply:
done

Reply:
I don't see the change in the function name

Reply:
oh, now